tools/package-chrome.sh

```diff
@@ -1,4 +1,5 @@
 set -e
 mkdir -p /opt/google/chrome
+apk update
 apk add chromium
```

Comment:
To be optimal, I think this should just be 
```suggestion
apk add --no-cache chromium
```